lms/djangoapps/certificates/tests/test_tasks.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+import ddt
+from mock import patch
+
+from lms.djangoapps.certificates.tasks import generate_certificate
+
+
+@ddt.ddt
+class GenerateUserCertificateTest(TestCase):
+    @patch('lms.djangoapps.certificates.tasks.generate_user_certificates')
+    def test_cert_task(self, generate_user_certs_mock):
+        generate_certificate(student='a', course_key='b', otherarg='c', otherotherarg='d')
```

Comment:
Nvm, I must have made that up then haha. Looks like we test the grades task using apply: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/tests/test_tasks.py#L360

Anywho, either way is still 👍 from me